{
  "database-name": "vectorstore",
  "webapp-cognito-domain-prefix": "threat-model",
  "acknowledged-issue-numbers": [
    31716,
    31716
  ]
}
